docs/quickstart/views/main.ejs

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Learnosity SDK - NodeJS</title>
+    <script src="//authoraide.dev.learnosity.com"></script>
```

Comment:
I think this should be as below even though it has CORS issue. but lets discuss this tomorrow in standup
```suggestion
    <script src="//authoraide.learnosity.com"></script>
```